fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaVersion.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.schema.control;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Tracks the incremental changes to the FHIR schema as it evolves. Incremental
+ * changes to the schema should be recorded here to create a new version number
+ * and this enum can then be used to identify the schema objects associated with
+ * a particular version.
+ */
+public enum FhirSchemaVersion {
+    
+    // Make sure the vid values are unique...this cannot be done programmatically with an enum
+     V0001(1, "Initial version")
+    ,V0002(2, "Composite search value support")
+    ,V0003(3, "issue-1263 fhir_ref_sequence start with 20000")
```

Comment:
Why should this be programmatically accessible?  Wouldn't a comment suffice?

Reply:
Useful in info/error messages to see which updates are being applied.